### PR TITLE
Members list

### PR DESCRIPTION
**Pull request by TylerK**
Stubbing in members list and re-usable avatars.

### CODE REVIEW

File: src/app/chat/members/members.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export default React.createClass({
+  displayName: 'MembersList',
+
+  propTypes: {
+    members: React.PropTypes.array.isRequired
+  },
+
+  getDefaultProps () {
+    return {
+      members: [
```
Comment (itsthatguy) [inline src/app/chat/members/members.jsx:14]: The adapter is not setup for this data structure, we need to transform it first

**Comment by itsthatguy** [comment]
Let's get this hooked up with: https://github.com/itsthatguy/switchboard/pull/22 so that we can merge into master, and called done

**Comment by itsthatguy** [comment]
Hooks up the proper data structure: https://github.com/itsthatguy/switchboard/commit/7af03c772fdbe74cd99cd321f8944b4d1fdbb0c3

**Comment by TylerK** [comment]
Can we merge this?

**Comment by itsthatguy** [comment]
:+1:

**Pull request merged**